readme/template-readme.md

```diff
@@ -0,0 +1,101 @@
+# README Template
+
+*{Before using this template, read the accompanying README template guide}.*
+
+## {Project} logo and badges
+{This section is optional. Share your logo and add relevant badges to your README.}
+
+## {Project Name} 
+{Optional: Include the project URL and the project owner name}
+
+## Table of contents 
+{This section is optional.}
+1. Heading 1
+2. Heading 2
+3. Heading 3...
+
+## Project description 
+{Project description example available in README template guide}
+
```

Comment:
**Suggestion (non-blocking):** I wonder if it would be helpful to indicate that these are just suggested sentence structures. Maybe if we said: {Here's some examples of effective phrases for describing a project: ..."